pipelines/src/test/groovy/IndividualBuildConfigTest.groovy

```diff
@@ -40,6 +43,7 @@ class IndividualBuildConfigTest {
                 ADDITIONAL_FILE_NAME_TAG   : 'k',
                 JDK_BOOT_VERSION           : 'l',
                 RELEASE                    : false,
+                WEEKLY                     : false,
```

Comment:
We need to add WEEKLY to the jenkins-helper definition: https://github.com/adoptium/ci-jenkins-pipelines/pull/727/files

Reply:
I really should read all the comments! It's here: https://github.com/adoptium/jenkins-helper/pull/51/files
Merged now